server/src/router/admin.ts

```diff
@@ -36,7 +38,10 @@ router.put('/:id', validateUser('Admin'), (req, res) => {
     params: { id },
     body,
   } = req;
-  db.update(res, Admin, { id }, body, tableName);
+
```

Comment:
Looks pretty good! One thing I would recommend is moving this condition to the update function in the `commons.ts` file so that it works for all the usage of updates across all the routes. Then, you won't have to use the conditional update function